### ISSUE TITLE
ORD Financial Perspectives changes

### ISSUE THREAD

**Issue by johcarter** [OasisLMF/ODS_OpenExposureData#37]
After reviewing the ORD Financial Perspectives, we have the following proposed changes (See attached)
[ord_perspectives_proposed.zip](https://github.com/OasisLMF/OpenDataStandards/files/6693799/ord_perspectives_proposed.zip)
The main proposed changes;

* Removing the following perspectives as standalone perspectives. 

3. Reinsurance FAC
4. Reinsurance QS
5. Reinsurance SS
6. Reinsurance PR
7. Reinsurance CXL

These can instead be supported by using 'reinsurance type' summary level breakdowns of the Reinsurance Loss perspective.

* Some new perspectives are added, and also a new reinsurance type Stop Loss 'SL' (not yet defined in OED)
* Descriptions for each perspective added and some context around summary level breakdowns of perspectives.

**Comment by MattDonovan82** [comment]
@johcarter could these be added to v2.0.0?

**Comment by johcarter** [comment]
No, I think it should we should run it against the steering committee first @MattDonovan82

**Comment by MattDonovan82** [comment]
ok, I'll make sure its discussed at the next SC meeting

**Comment by MattDonovan82** [comment]
@johcarter I'll add this to the agenda for the SC - are you ok to attend and discuss this on 20th Sept?

**Comment by MattDonovan82** [comment]
closed as included in ODS v2.0.0

**Comment by johcarter** [comment]
Reopening as not implemented in ODS_OpenResultsData. Issue needs transferring @MattDonovan82
